src/server/browserContext.ts

```diff
@@ -74,7 +74,8 @@ export abstract class BrowserContext extends SdkObject {
     this._closePromise = new Promise(fulfill => this._closePromiseFulfill = fulfill);
 
     if (this._options.recordHar)
-      this._harTracer = new HarTracer(this, this._options.recordHar);
+      this._harTracer = new HarTracer(this, {...this._options.recordHar, path: path.join(this._browser.options.artifactsDir, `${createGuid}.har`)});
```

Comment:
I don't think this does what you think this does, which suggests we might be missing a test with 2 hars.

Reply:
Oops! Missed the actual invocation!